<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SamplePlugin</class>
 <widget class="QDockWidget" name="SamplePlugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="_btn0">
        <property name="text">
         <string>Load background</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn1">
        <property name="text">
         <string>StartTimer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn2">
        <property name="text">
         <string>Open saving files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn3">
        <property name="text">
         <string>Read data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn4">
        <property name="text">
         <string>Initialize</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="_checkBox">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="_spinBox"/>
      </item>
      <item>
       <widget class="QSlider" name="_slider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="_label">
        <property name="text">
         <string>Label</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="_label_marker">
        <property name="text">
         <string>Label_containing_marker</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
